src/main/java/com/github/tomakehurst/wiremock/http/CaseInsensitiveKey.java

```diff
@@ -39,9 +39,7 @@ public boolean equals(Object o) {
 
     CaseInsensitiveKey that = (CaseInsensitiveKey) o;
 
-    if (key != null ? !key.equalsIgnoreCase(that.key) : that.key != null) return false;
-
-    return true;
+    return key == null ? that.key == null : key.equalsIgnoreCase(that.key);
```

Comment:
personally, I'd prefer this to be:

```suggestion
    if (key == null) {
      return that.key == null;
    } else {
      key.equalsIgnoreCase(that.key);
    }
```

Reply:
```suggestion
    if (key == null) {
      return that.key == null;
    }
    return key.equalsIgnoreCase(that.key);
```
You were missing a return statement; would you be OK with this instead? I don't like unnecessarily nested else statements when we can avoid them with a return.

Reply:
Woops, good spot.

Yep, I like that too, that'd be appreciated 👍🏽